prediction_server/app.py

```diff
@@ -18,30 +18,38 @@
 from classifiers import SimpleClassifier
 from pyod.models.hbos import HBOS
 
+from str_helpers import *
+
 app = Flask(__name__)
 
 @app.route('/init', methods=['GET'])
 def init():
     """ Initialize predictor """
     # move old file
-    if os.path.isfile(config.data_filename) and os.stat(config.data_filename).st_size > 0:
-        filename = os.path.splitext(config.data_filename)[0]
-        filename += datetime.datetime.now().strftime("_%Y%m%d_%H%M%S")
-        filename += ".csv"
-        shutil.move(config.data_filename, filename)
+    device_uuid = request.args.get('deviceUUID')
 
-    # delete model file
-    if os.path.isfile(config.model_filename):
-        os.remove(config.model_filename)
+    date_time = datetime.datetime.now().strftime("%Y%m%d_%H%M%S")
+    data_filename = get_data_filename(device_uuid, date_time)
 
-    return jsonify({"status" : 0})
+    new_data_file = open(data_filename, "w+")
+    new_data_file.close()
+
+    device_model_folder = config.model_filepath + device_uuid
+    if(os.path.exists(device_model_folder)):
+        for model_file in os.listdir(device_model_folder):
+            os.remove(device_model_folder+"/" + model_file)
```

Comment:
This is a bit dangerous, please verify that device_uuid exists and is not empty in the lines above, as giving an empty deviceUUID could delete everyfile under models/

Reply:
Thanks eyal, that's very true.